server/views.go

```diff
@@ -53,3 +54,55 @@ func handleAddPlayer(ctx *HandlerCtx) http.HandlerFunc {
 		}
 	}
 }
+
+func handleAddScoreSelectPlayers(ctx *HandlerCtx) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case "GET":
+			players := ctx.DB.GetAllPlayers()
+			ctx.Template.Render(w, players)
+		case "POST":
+			r.ParseForm()
+			addScoreURL, err := url.Parse("/score/add/")
+			if err != nil {
+				panic(err)
+			}
+			q := addScoreURL.Query()
+			for name := range r.Form {
+				q.Add("player", name)
+			}
+			addScoreURL.RawQuery = q.Encode()
+			http.Redirect(w, r, addScoreURL.String(), 302)
+		}
+	}
+}
+
+func handleAddScore(ctx *HandlerCtx) http.HandlerFunc {
+	categories := []string{
+		"Military Conflicts",
+		"Wonders",
+		"Treasury Contents",
+		"Civilian Structures",
+		"Scientific Structures",
+		"Commercial Structures",
+		"Guilds",
+	}
+	return func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case "GET":
+			players := r.URL.Query()["player"]
+			ctx.Template.Render(w, struct {
```

Comment:
This is a little unreadable, consider using something like:
```
var dataWrapper struct {
	Players []string
	Categories []string
}

dataWrapper.Players = players
dataWrapper.Categories = categories
```

Reply:
done